docs/tutorials/NMR/Example_1_-Ethanol.md

```diff
@@ -0,0 +1,115 @@
+ |                      Chemical Shielding Tensor                   |
+ |------------------------------------------------------------------|
+ |     Nucleus                            Shielding tensor          |
+ |  Species            Ion            Iso(ppm)   Aniso(ppm)  Asym   |
+ |    H                1               29.45       8.84      0.14   |
+ |    H                2               30.10       8.07      0.20   |
+ |    H                3               29.94       7.12      0.06   |
+ |    H                4               26.83       8.02      0.95   |
+ |    H                5               27.24      -7.07      0.90   |
+ |    H                6               31.93      13.99      0.46   |
+ |    C                1              157.27      33.77      0.70   |
+ |    C                2              110.73      69.91      0.42   |
+ |    O                1              268.63     -50.78      0.96   |
+ ====================================================================
+ ```
+ Here we are only interested in the isotropic values
+
+
+3. This information, plus the full tensors, are also given in the file ethanol.magres
+
+4. You might wish to transfer the *.magres file back to your desktop to visualise with [MagresView](https://www.ccpnc.ac.uk/magresview/magresview/magres_view.html?JS).
+
+<ol start="5"> <li> Identify which (hydrogen) ion corresponds to which part of the molecule - in the case of ethanol, find out which ones correspond to CH<sub>3</sub>, CH<sub>2</sub> and OH.
+   This can be done by uploading the cell file to VESTA and clicking each atom.
+   <br>
+   <img alt="NMR vesta demonstration" src="../../img/NMR_vesta_demonstration.png" width="400"/>
+   <br>
+   Here you can see that one of the CH<sub>3</sub> hydrogens is atom 3
+</li>
+<li> Examine the effect
+ of increasing the cutoff energy (say 20-50 Ryd in steps of 10 Ryd). It always helps to plot a graph of the convergence (e.g. with gnuplot or xmgrace on the cluster - or with excel on the PC). The result looks a bit like this: <br> <img alt="Ethanol convergence plot" src="../../img/ethanol_convergence.png" width="500"/>
+</li>
+<li> Find the "converged" hydrogen (or proton in NMR language) shieldings. We will compare them to experiment. The three methyl (CH<sub>3</sub>) protons undergo fast exchange; they "rotate" faster than the nuclear magnetic moment processes. The magnetic moment will therefore "see" an average chemical shielding. The same is true of the CH<sub>2</sub> protons.
+</li>
+<li> Average the CH<sub>3</sub> and CH<sub>2</sub> chemical shieldings. This will give you 3 unique chemical shieldings (including the OH reading).
+</li>
+<li> We now need to convert the chemical shieldings &#963;<sub>iso</sub> to chemical shifts &#948;<sub>iso</sub> on the experimental scale. We use the relation:  <i>&#948;<sub>iso</sub>=&#963;<sub>ref</sub>-&#963;</i>
+<br>
+
+A suitable &#963;<sub>ref</sub> for <sup><small>1</small></sup>H is 30.97ppm.
```

Comment:
Superscript in markdown can be `^x^`

```suggestion
A suitable $\sum_{ref}$ for ^1^H is 30.97ppm.
```

Reply:
That's weird:
![28-06-2024:09:44:51](https://github.com/castep-docs/castep-docs/assets/46597752/1b8be326-2dc3-4a9e-b408-e2ce5ca2d76f)
Seems to render for me.

Reply:
The reason it didn't for me was because I tested it insdie a HTML <figure><figcaption> object. 

I'll look into doing figure captions in HTML later, though in that case we might wanna change the NEB tutorial because that's the way it's done there

Reply:
Although it should be a lower case sigma (in latex \sigma_{ref}) not a sum

Reply:
Ah, sorry, I just quickly googled `&#963;`, and it came up with sigma, so I assumed it was meant to be a sum.